frontend/src/component/admin/apiToken/CreateApiToken/CreateApiToken.tsx

```diff
@@ -22,17 +25,26 @@ import {
     CREATE_CLIENT_API_TOKEN,
     CREATE_FRONTEND_API_TOKEN,
 } from '@server/types/permissions';
+import { Limit } from 'component/common/Limit/Limit';
 
 const pageTitle = 'Create API token';
 interface ICreateApiTokenProps {
     modal?: boolean;
 }
+
+const StyledLimit = styled(Limit)(({ theme }) => ({
+    margin: theme.spacing(2, 0, 4),
+}));
```

Comment:
As we discussed after standup yesterday, I don't love that we style the limit component *for* the form _outside_ the form. However, this form also appears to be different than the others (it's got more children and fewer props), so it may be a worthwhile tradeoff.